using-s3-source.md

```diff
@@ -45,17 +52,19 @@ Add your s3 source under `:sources` key. Each s3 bucket should have its own set
 
 Or set AWS access id, secret and session token explicitly.
 
-> Note that you need to add `<path_to_gems_dir>` to your s3 source uri, if your gem repo doesn't exist at the root of the bucket.
+> Note that you need to add `<path_to_gems_dir>/` to your s3 source uri, if your gem repo doesn't exist at the root of the bucket. NOTE: The trailing slash.
 
     $ cat ~/.gemrc
-    ....
     :sources:
+    - s3://my-great-bucket/my-gems/ 
     - s3://bucket1/
-    - s3://bucket2/
-    - s3://bucket3/path_to_gems_dir
-    - s3://bucket4/
```

Comment:
Can we please fix the bucket3 line instead of adding a new one? Also, let's not remove bucket2 and bucket4, they are used to show various ways to configure s3_source, see lines 63 and 72.